aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -132,6 +150,15 @@ else
 	makePost "adminrules" '{"name":"core-user-rule","description": "Core User is Admin of all containers","role":"administrator","resources":{"containers":["*"],"images":["*"],"volumes":["*"],"networks":["*"]},"accessors":{"users":["core"]}}'
 fi
 
+# See if aqua qualys integration already exists
+EXISTING_QUALYS=$(makeGet settings/integrations/qualys)
+
+if [[ "$EXISTING_QUALYS" == "200" ]]; then
+	log "qualys integration exists..."
+else
+	makePost "settings/integrations/qualys" '{ "enabled":true, "url":"$QUALYS_URL", "username":"$QUALYS_USERNAME", "password":"$QUALYS_PASSWORD" }'
```

Comment:
This won't work because you're using bash variables inside of single quotes. See [this line](https://github.com/anshupande/dcos-scripts/blob/b4d79cfb6d32b62f8f349c1e6a68498c20430889/aqua-ethos-setup/aqua_ethos.sh#L167) for an example of how to get around that